core/message-tags-3.3.md

```diff
@@ -80,6 +88,36 @@ Individual tag keys MUST only be used a maximum of once per message. Clients
 receiving messages with more than one occurrence of a tag key SHOULD discard all
 but the final occurrence.
 
+### The `TAGMSG` tag-only event
+
+A new event `TAGMSG` is defined for sending messages with tags but no text content.
+This event MUST be delivered to targets in the same way as `PRIVMSG` and `NOTICE`
+events, taking into account channel membership and modes.
```

Comment:
>Is there a statusmsg spec we can refer to?

It's mentioned very briefly in the [ISUPPORT drafts](https://tools.ietf.org/html/draft-hardy-irc-isupport-00#section-4.18).

I'd say it counts as behaving exactly the same as PRIVMSG/NOTICE, but explicit mentions don't hurt.